src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -506,4 +506,456 @@ select
         measure_level_met_district_bm_pm_period_total_students_assessed,
         measure_level_met_district_bm_pm_period_total_students_enrolled
     ) as measure_level_met_district_bm_pm_period_percent,
-from with_counts
+from with_counts*/
+
+
+
+{% set periods = ["Boy", "Boy->moy", "moy", "moy->eoy", "eoy"] %}

```

Comment:
the case has changed from above, is that correct?

Reply:
I had already fixed this. Blargh!